assemblers/org.osbuild.qemu

```diff
@@ -196,6 +204,8 @@ class Filesystem:
             maker = mkfs_xfs
         elif fs_type == "vfat":
             maker = mkfs_vfat
+        elif fs_type == "btrfs":
+            maker = mkfs_vfat
```

Comment:
```suggestion
            maker = mkfs_btrfs
```

Reply:
Thanks, I've pushed new version with this fixed. And yeah, happy to add test coverage but I wasn't sure where/how to.